### ISSUE TITLE
Preview on existing projects is broken

### ISSUE THREAD

**Issue by ghost** [justintadlock/custom-content-portfolio#12]
When editing an existing project and you want to preview your changes before updating, I get a 404 error page.
When creating a new project the preview function is working properly, as long as you don't publish it.

**Comment by justintadlock** [comment]
This is working fine for me on WP 4.4.  Can you provide precise steps to reproduce?  Does this happen with other post types?

**Comment by tvdheijden** [comment]
Steps to reproduce:
1. Open an existing project with status "Published" for editing
2. Press the Preview button in the post editor
3. A new tab opens displaying a 404 error page

This does not happen with other post types (default WP post and pages), for these post types the preview function works properly. I noticed that when I change the project status to "draft", save the project and then preview it again, it works perfectly well. Change the status back to "published" again, and it's broken.

Next to that, I noticed when editing and previewing an existing post or page the preview URL ends with `&preview=true`. When previewing an existing and published portfolio project, the `&preview=true` is missing from the preview URL. Again, when changing the project status to "draft", saving and previewing, the `&preview=true` is at the end of the preview URL.

I am experiencing the same behaviour on two sites with WP 4.4.
